src/game/protos/object.c

```diff
@@ -568,7 +570,10 @@ void object_set_animation(object *obj, animation *ani) {
 
     // Debug texts
     if(obj->cur_animation->id == -1) {
-        log_debug("Custom object set to (x,y) = (%f,%f).", obj->pos.x, obj->pos.y);
+        char bufx[FIXEDPT_STR_BUFSIZE], bufy[FIXEDPT_STR_BUFSIZE];
+        fixedpt_str(obj->pos.fx, bufx, -1);
+        fixedpt_str(obj->pos.fy, bufy, -1);
+        log_debug("Custom object set to (x,y) = (%s,%s).", bufx, bufy);
```

Comment:
Fixed point numbers can be formatted without conversion to floating point, but it's a bit verbose-- and fixedpt_str is a scary function (it writes to a string with no bounds checking-- like its 1982, despite the library being written in the modern year of 2012)